tsconfig.json

```diff
@@ -3,13 +3,13 @@
     "skipLibCheck": false,
     /* Basic Options */
     // "incremental": true,                   /* Enable incremental compilation */
-    "target": "ESNext",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
-    "module": "ESNext",                       /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
+    "target": "ES6",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
+    "module": "ES6",                       /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
```

Comment:
"module" option describes used module syntax (are those es2015 modules with imports, commonjs requires or what else), while "target" describes the assumed level of features at the target runtime (say an old browser would require ES3)

With that in mind, i believe we should have `"target: "ES2019", "module": "es2015"`.

I believe the issue you had is related to `"ES6"` not being a valid module format, so it defaults to `"commonjs"`.

Reply:
At least on my computer, I can't run ts-node used to run the benchmarks without having "module": "commonJS" enabled. I get the infamous: SyntaxError: Cannot use import statement outside a module. Any clue?

Reply:
Target has those available values from VSCode:

- Specify ECMAScript target version: 'ES3', 'ES5', 'ES6'/'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', 'ESNext'

- Specify module code generation: 'None', 'CommonJS', 'AMD', 'System', 'UMD', 'ES6', 'ES2015', 'ES2020' or 'ESNext'. Only 'AMD' and 'System' can be used in conjunction with --outFile.

Reply:
try adding `"type": "module"` to `package.json`